src/Parsing/Impl/Ast/FunctionDefinition.cs

```diff
@@ -19,21 +19,18 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Python.Core;
-using Microsoft.Python.Core.Text;
 
 namespace Microsoft.Python.Parsing.Ast {
     [DebuggerDisplay("{Name}")]
-    public class FunctionDefinition : ScopeStatement, IMaybeAsyncStatement {
+    public class FunctionDefinition : ScopeStatement, IMaybeAsyncStatement, IScopeNode {
```

Comment:
I assume `ScopeStatement` implements `IScopeNode` so `IScopeNode` here is unnecessary.